src/AppInstallerCLI.sln

```diff
@@ -145,20 +150,26 @@ Global
 		ManifestSchema\ManifestSchema.vcxitems*{fb313532-38b0-4676-9303-ab200aa13576}*SharedItemsImports = 4
 	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU

```

Comment:
We don't build "Any CPU", so I'm not sure how this is going to work.

We do have C# projects that are built without it though; if there are problems you should be able to follow the E2E tests (among others).